layouts/_default/main.html

```diff
@@ -4,122 +4,51 @@
+    {{ with .Page.Params.subhed }}
+      <h2 class="font-black text-5xl mb-5 uppercase">{{ . }}</h2>
+    {{ end }}
+    {{ with .Content }}
+      <div class="article-content max-w-2xl mx-auto text-lg">
+        {{ . }}
       </div>
-
-      {{ with .Param "authors" }}
-        <footer class="block" data-ga-category="inline-footer:author">
-          {{ range $name := . }}
-            {{ $author := partial "helper/get-author" $name }}
-            <div class="level">
-              <div class="level-left">
-                <div class="level-item">
-                  {{ if $author.thumb }}
-                    {{ if $author.link }}
-                      <a href="{{ $author.link }}" class="image is-64x64">
-                        <img
-                          src="{{ $author.thumb }}"
-                          alt="Photo of {{ $author.name }}"
-                          class="is-rounded is-monochrome"
-                          loading="lazy"
-                          decoding="async"
-                        />
-                      </a>
-                    {{ else }}
-                      <span class="image is-64x64">
-                        <img
-                          src="{{ $author.thumb }}"
-                          alt="Photo of {{ $author.name }}"
-                          class="is-rounded is-monochrome"
-                          loading="lazy"
-                          decoding="async"
-                        />
-                      </span>
-                    {{ end }}
-                  {{ else }}
-                    <span class="image is-64x64"></span>
-                  {{ end }}
-                </div>
-
-                <div class="level-item">
-                  <div>
-                    <h3 class="subtitle is-3 has-text-weight-semibold">
-                      {{ if and $author.active $author.link }}
-                        <a href="{{ $author.link }}">
-                          {{ $author.name }}
-                        </a>
-                      {{ else }}
-                        <span class="has-text-grey">
-                          {{ $author.name }}
-                        </span>
-                      {{ end }}
-                    </h3>
-                    {{ if and $author.active $author.title }}
-                      <h3 class="subtitle ">
-                        {{ $author.title }}
-                      </h3>
-                    {{ end }}
-                  </div>
-                </div>
-              </div>
-            </div>
-          {{ end }}
-        </footer>
-      {{ end }}
+    {{ end }}
+  </div>
+  {{ with .Page.RenderString (.Param "newsletter") }}
+    <div class="-mx-4 font-sans md:mx-0">
+      {{ . }}
     </div>
-  </article>
+  {{ end }}
+
+  {{ range $i, $blocks := .Params.blocks }}
+    {{ $id := urlize .title }}
+    <div
+      id="{{ $id }}"
+      x-data
+      x-intersect.half="$dispatch('intersect:support', $el.id)"
```

Comment:
I think this is for the sticky menu on the about page. Do we need it elsewhere?

Reply:
I've removed this: https://github.com/spotlightpa/poor-richard/pull/368/commits/08f696ce271a7809053fd11f3d6ebd26b8fb11b4